app/Models/Leaderboard.php

```diff
@@ -157,6 +158,75 @@ public function entries(bool $includeUnrankedUsers = false): HasMany
             });
     }
 
+    /**
+     * @return HasMany<LeaderboardEntry>
+     */
+    public function sortedEntries(): HasMany
+    {
+        $entries = $this->entries();
+
+        if ($this->Format === ValueFormat::ValueUnsigned) {
+            if ($this->LowerIsBetter) {
+                $entries->orderByRaw(toUnsignedStatement('score'));
+            } else {
+                $entries->orderByRaw(toUnsignedStatement('score') . ' DESC');
+            }
+        } elseif ($this->LowerIsBetter) {
+            $entries->orderBy('score');
+        } else {
+            $entries->orderByDesc('score');
+        }
```

Comment:
`orderBy()` accepts a second parameter to specify the direction. We may be able to make this more concise:

```php
$direction = $this->LowerIsBetter ? 'ASC' : 'DESC';

if ($this->Format === ValueFormat::ValueUnsigned) {
    $entries->orderByRaw(toUnsignedStatement('score') . " $direction");
} else {
    $entries->orderBy('score', $direction);
}
```

Reply:
done